Gemfile

```diff
@@ -30,4 +26,7 @@ group :development do
   gem 'guard-rspec'
   gem 'guard-rubocop'
   gem 'mixlib-versioning'
+  gem 'mixlib-install', github: 'chef/mixlib-install'
```

Comment:
Is there not a version released on rubygems.org to use? Should we pin this to a specific shasum?